.pre-commit-config.yaml

```
@@ -60,7 +60,11 @@ repos:
           - mdformat-black
           - mdformat_frontmatter
 
-  - repo: https://github.com/PyCQA/flake8
-    rev: 6.0.0
+  - repo: https://github.com/charliermarsh/ruff-pre-commit
+    rev: v0.0.226
```

maybe use the latest release?
```suggestion
    rev: v0.0.233
```

---

Sure, it is just moving so fast...